includes/polling/ports.inc.php

```diff
@@ -363,6 +363,54 @@
     }
 }
 
+//NAC Polling
+if ($config['enable_ports_nac']) {
```

Comment:
I think this should be it's own module instead of a port of the ports poller.

Reply:
It was my first idea, but discussing on Discord, laf said something about use the actual port structure.

laf08/30/2018
So that might be something better to be part of the existing ports polling

Jose Augusto Brazil [<o>]08/30/2018
yes, could be.. but if I do that, I think will not possible disable this polling if somebody doesn't use NAC, and maybe will increase the polling time... or not?

laf08/30/2018
That depends on how much additional information you need to collect but that can be controlled by a config option